packages/atlas/src/providers/videoWorkspace/provider.tsx

```diff
@@ -24,7 +24,7 @@ export const VideoWorkspaceProvider: FC<PropsWithChildren> = ({ children }) => {
   }, [])
 
   // trigger categories fetch so that they are available once the video workspace is opened
-  useCategories(undefined, {
+  useQnCategories(undefined, {
     context: { delay: 1500 },
     onError: (error) => SentryLogger.error('Failed to fetch video categories', 'VideoWorkspaceProvider', error),
   })
```

Comment:
This probably won't be needed anymore once we use display categories in video workspace